cpuinfo_linux.go

```diff
@@ -158,3 +159,40 @@ func getCPUVariant() (string, error) {
 
 	return variant, nil
 }
+
+func getAmd64MicroArchLevel() (string, error) {
+	flags, err := getCPUInfo("flags")
+	if errors.Is(err, errNotFound) {
+		return "", fmt.Errorf("failure getting CPU flags: %v", err)
+	}
+
+	containsAll := func(set map[string]interface{}, toMatch []string) bool {
+		for _, m := range toMatch {
+			if _, ok := set[m]; !ok {
+				return false
+			}
+		}
+		return true
+	}
+
+	flagSet := map[string]interface{}{}
+	for _, flag := range strings.Split(flags, " ") {
+		flagSet[flag] = true
+	}
+
+	// https://unix.stackexchange.com/questions/631217/how-do-i-check-if-my-cpu-supports-x86-64-v2
```

Comment:
It may be better to reference [`https://tip.golang.org/wiki/MinimumRequirements#amd64`](https://tip.golang.org/wiki/MinimumRequirements#amd64) instead